### ISSUE TITLE
Add Httpstan Compatibility Chart to Documentation

### ISSUE THREAD

**Issue by AlexanderNenninger** [stan-dev/pystan#397]
#### Describe the problem with the documentation

Currently, the latest httpstan release is [4.11.0](https://github.com/stan-dev/httpstan/tree/4.11.0) while the latest supported httpstan version is `4.10.1`. This is not apparent in the instructions for installing pystan from source.

#### Suggest a potential alternative/fix

Add specific httpstan version requirements to documentation.

**Comment by riddell-stan** [comment]
Good point. The normal release process got delayed a bit. This should be resolved soon.
